pkg/ccl/storageccl/import.go

```diff
@@ -33,9 +33,53 @@ var importBatchSize = func() *settings.ByteSizeSetting {
 	s := settings.RegisterByteSizeSetting(
 		"kv.import.batch_size",
 		"the maximum size of the payload in an AddSSTable request",
+		16<<20,
+	)
+	return s
+}()
+
+var importPKAdderBufferSize = func() *settings.ByteSizeSetting {
+	s := settings.RegisterByteSizeSetting(
+		"kv.import.pk_buffer_size",
```

Comment:
`kv.import` confusingly does not refer to `IMPORT` but rather to `ImportRequest` which is part of `Restore`, but then even more confusingly is sometimes used by IMPORT.

That said, I don't have a _better_ suggestion for you:
  - there are `bulk_io` settings but they're also under `kv.`.
  - I could see a `sql.import` prefix making sense, but then if we have low-level settings for IMPORT they might not make sense there.
 - `bulk.import`  makes the most sense to me, but I donno if anyone would be annoyed about claiming new top-level prefixes.

I donno who has opinions on this. I used to be the person people asked about settings stuff, but I also tried really hard to stay out of naming bike sheds so I'm now out of the loop.

Reply:
@rolandcrosby would you be able to provide some input here? In my opinion `bulk.import` does make sense considering it is not really KV level values we are tweaking. If introducing a new top-level is an issue though what do you think about `kv.bulk_io_import`?

Reply:
The presence of the term "import" in these has tripped me up in the past. Maybe `kv.bulk_ingest` or something?